best-practices/development-tools/azure/devops/continuous-integration/azure-pipelines-azure-app-functions-ci.yml

```diff
@@ -0,0 +1,89 @@
+# .NET Core Function App to Windows on Azure
+# Build a .NET Core function app and deploy it to Azure as a Windows function App.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- main
+
+variables:
+  # Agent VM image name
+  vmImageName: 'vs2017-win2016'
+
+  # Working Directory
+  workingDirectory: '**'
+
+  TestFilesPath: '$(System.DefaultWorkingDirectory)\<testing-folder>\bin\$(buildConfiguration)\netcoreapp3.1'
+
+stages:
```

Comment:
I like the split between build and test in the pipelines here. Very nice.

Reply:
Thanks.